spec/features/add_iiif_manifest_spec.rb

```diff
@@ -29,6 +29,7 @@
   end
 
   it 'returns an error message if the URL returned in not a IIIF endpoint' do
+    pending
```

Comment:
Should we remove this now that projectblacklight/spotlight#1580 is merged or does this have to wait until a spotlight release?